src/Language/PureScript/Interactive/Directive.hs

```diff
@@ -93,14 +94,16 @@ hasArgument _ = True
 --
 help :: [(Directive, String, String)]
 help =
-  [ (Help,    "",         "Show this help menu")
-  , (Quit,    "",         "Quit PSCi")
-  , (Reload,  "",         "Reload all imported modules while discarding bindings")
-  , (Clear,   "",         "Discard all imported modules and declared bindings")
-  , (Browse,  "<module>", "See all functions in <module>")
-  , (Type,    "<expr>",   "Show the type of <expr>")
-  , (Kind,    "<type>",   "Show the kind of <type>")
-  , (Show,    "import",   "Show all imported modules")
-  , (Show,    "loaded",   "Show all loaded modules")
-  , (Paste,   "paste",    "Enter multiple lines, terminated by ^D")
+  [ (Help,     "",          "Show this help menu")
+  , (Quit,     "",          "Quit PSCi")
+  , (Reload,   "",          "Reload all imported modules while discarding bindings")
+  , (Clear,    "",          "Discard all imported modules and declared bindings")
+  , (Browse,   "<module>",  "See all functions in <module>")
+  , (Type,     "<expr>",    "Show the type of <expr>")
+  , (Kind,     "<type>",    "Show the kind of <type>")
+  , (Show,     "import",    "Show all imported modules")
+  , (Show,     "loaded",    "Show all loaded modules")
+  , (Paste,    "paste",     "Enter multiple lines, terminated by ^D")
+  , (Complete, "<(expr?)>", "Show completion output as if pressing tab")
```

Comment:
Just a minor nitpick, but prefix seems more accurate than expr here.

Reply:
I liked it, so went for it. Change is in now!